frontend/components/organiser/forms/FormEditor.tsx

```
@@ -0,0 +1,367 @@
+        </Tooltip>
+      </div>
+      <div className="block sm:hidden">
+        <Tooltip content={content} placement="top">
+          {children}
+        </Tooltip>
+      </div>
+    </div>
+  );
+};
+
+const FormEditor = ({}: FormEditorParams) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)